### ISSUE TITLE
-G option not working for consumer group 

### ISSUE THREAD

**Issue by bluedog13** [edenhill/kafkacat#327]
I see the below error when the consumer group is not added to the ACL
**_Consumer error: FindCoordinator response error: Group authorization failed._**

But after adding the consumer group to ACL and running the same command, I see the below error.  (via a client app, I can confirm this works as well)
**_ERROR: Topic -e [0] error: Subscribed topic not available: -e: Broker: Topic authorization failed_**

The Kafkacat command looks like below

```
kafkacat -C -b *******:9092 \
    -X security.protocol=sasl_ssl -X sasl.mechanism=PLAIN \
    -X sasl.username=*** -X sasl.password=*****  \
    -G zgroup my-topic \     #( NOTE: -t my-topic. works just fine when used instead of -G) 
    -e \
    -f '\nKey (%K bytes): %k
    Value (%S bytes): %s
    Timestamp: %T
    Partition: %p
    Offset: %o
    Headers: %h\n' 
```

**Comment by edenhill** [comment]
The topic(s) to subscribe to must be the last arguments.

**Comment by bluedog13** [comment]
Thank you.  

That seemed to work and I was able to connect to the topic using the consumer group.  However, I am not able to see the read messages.  Is there something I am missing to see the read messages?

Below is the change I made

`kafkacat -C -b *******:9092 \
    -X security.protocol=sasl_ssl -X sasl.mechanism=PLAIN \
    -X sasl.username=*** -X sasl.password=*****  \
    -e \
    -f '\nKey (%K bytes): %k
    Value (%S bytes): %s
    Timestamp: %T
    Partition: %p
    Offset: %o
    Headers: %h\n' 
   - G zgroup my-topic 

Output. (I see that it reaches the end for each partition but does not exist as well)

% Waiting for group rebalance
% Group zgroup rebalanced (memberid rdkafka-a9a3d34f-4834-4886-9c78-9e72cfc7f74e): assigned: zsamples-sampleapp-test-sandbox-private [0], zsamples-sampleapp-test-sandbox-private [1], zsamples-sampleapp-test-sandbox-private [2]
% Reached end of topic zsamples-sampleapp-test-sandbox-private [1] at offset 4
% Reached end of topic zsamples-sampleapp-test-sandbox-private [0] at offset 4
% Reached end of topic zsamples-sampleapp-test-sandbox-private [2] at offset 3
